### ISSUE TITLE
No incoming http request at terminal

### ISSUE THREAD

**Issue by namhq1989** [developit/express-es6-rest-api#21]
Hi,
With my old project, I see the incoming http requests at terminal like this:
```
POST /api/login 200 816 - 42.802 ms
POST /api/users/me 200 800 - 39.342 ms
```
But when I run this boilerplate, I see nothing at terminal, so can someone show me the way to enable terminal logs again? Very thanks!

My run script:
```
"scripts": {
    "dev": "nodemon -w server --verbose --exec 'NODE_ENV=development DEBUG=App,$DEBUG babel-node index.js --presets es2015,stage-3'",
    "start": "DEBUG=App,$DEBUG babel-node index.js",
    "test": "NODE_ENV=test mocha --trace-deprecation server/test/$file --compilers js:babel-core/register --timeout 20000 --slow 500"
  }
```

**Comment by zouhir** [comment]
I'll have sometime in my hand this evening to investigate further. However, as something on top of my head: Express little logger as a middlewear might unblock you:

```js
var logger = function(req, res, next) {
    console.log(`Logging the request: ${req.url}`);
    console.log(`Logging the respone status: ${res.status}`);
    next(); // Passing the request to the next handler in the stack.
}
```

I'll make sure I test & investigate the issue for you asap!

**Comment by namhq1989** [comment]
Thank you for quick reply, but I added morgan instead of use express.logger (I read this in [Moving to Express 4](https://expressjs.com/en/guide/migrating-4.html)). And very pleasure to hear something good about this from you.
Thanks!

**Comment by developit** [comment]
@namhq1989 Morgan would work.  If you have a solution you like, I'd be happy to accept a PR that adds logging to this boilerplate by default.  Morgan and its default Express integration would be nice.

**Comment by namhq1989** [comment]
@developit I added it, please check and update your code.
Thanks!

**Comment by developit** [comment]
Fixed after merging #22 :)
